.github/workflows/pr-opened.yml

```
@@ -0,0 +1,31 @@
+on:
+  # Run for PRs to main
+  pull_request:
+    branches:
+    - main
+    types: [ opened, synchronized ]
+
+
+jobs:
+  # Detect any changes to Docker files
+  docker-check:
+    runs-on: ubuntu-latest
```

actionlint (1.7.4)

when a reusable workflow is called with "uses", "runs-on" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "docker-check"

(syntax-check)